dotcom-rendering/src/web/server/index.ts

```diff
@@ -181,14 +181,14 @@ export const renderKeyEvents = (
 };
 
 export const renderFront = (
-	{ body, query }: express.Request,
+	{ body }: express.Request,
 	res: express.Response,
 ): void => {
 	try {
 		const enhanced = enhanceFront(body);
```

Comment:
Maybe this could be called `Front`, and maybe we could keep it lowercase?

Reply:
Agreed. Both on dropping 'enhanced' and on using lowercase 👍